src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -805,9 +805,7 @@ qreal AbstractNotationPaintView::startVerticalScrollPosition() const
 bool AbstractNotationPaintView::adjustCanvasPosition(const RectF& logicRect, bool adjustVertically)
 {
     TRACEFUNC;
-
     RectF viewRect = viewport();
-
```

Comment:
Please preserve these empty lines (here and also before `return moveCanvasToPosition(pos);` both times)